lib/env_bang.rb

```diff
@@ -24,8 +24,11 @@ def use(var, *args)
     end
 
     def raise_formatted_error(var, description)
+      return if ENV['SKIP_ENV_ERRORS']
```

Comment:
Let's name it `ENV_BANG` and check against `"false"` string to be more explicit and readalble during usage:
```
return if ENV['ENV_BANG'] == 'false'
```